documentation/howto/tag-and-release-role.md

```diff
@@ -31,20 +31,26 @@ This includes the git tag - the git tag should be **identical** to the version
 number - do not use git tags like `v1.1.0` or `1.1.0-rc` - it must be strictly
 `X.Y.Z`.
 
-## Create Git Tag and Release on Github
+## Update changelogs, Create Git Tag, and Release on Github
 
-Go to the page `https://github.com/linux-system-roles/$ROLENAME/releases/new`
-e.g. https://github.com/linux-system-roles/network/releases/new
+Script [role-make-version-changelog.sh](https://github.com/linux-system-roles/auto-maintenance#role-make-version-changelogsh) is used to create a new version,
+tag, and release for a role. It will guide you through the process.
+It will show you the changes in the role since the last tag, and ask you
+what will be the new semantic version to use for the tag. It will then put
+the changes in a file to use for the update to the CHANGELOG.md file for
+the new version, and put you in your editor to edit the file. If you are
+using this in conjunction with local-repo-dev-sync.sh, it will push the
+changes to your repo and create a pull request for CHANGELOG.md. Once the
+CHANGELOG.md PR is merged, there is github action automation to tag the
+repo with the version, create a github release, and import the new version
+into Ansible Galaxy.
 
-Fill in the version in `X.Y.Z` format - this will also create the git tag if you
-have not already done so.  Select the branch to tag (usually `main` or
-`master`).
+For an example of a good changelog and the corresponding release, see
+https://github.com/linux-system-roles/network/blob/main/CHANGELOG.md#1100---2022-11-01
+https://github.com/linux-system-roles/network/releases/tag/1.10.0
 
-Give the release a descriptive title, and provide release notes in the text
-field.
-
-For an example of a good release, see
-https://github.com/linux-system-roles/network/releases/tag/1.3.0
+The example network [1.10.0] shows all three sections have one or more items.
+If there are no items in a section, omit the section.
 
 ## Publish on Ansible Galaxy
```

Comment:
We should change this section.  I think the `Import` button only works if the role main branch name is `master`.  I think we should say something like this:
```
The github action automation should do everything for you - tag, github repo release, and publish to Galaxy.

If for some reason you need to manually publish a role, you must first ensure the role is tagged and there is a github release.  Then, use

ansible-galaxy role import -vv --branch main --role-name linux-system-roles $ROLENAME

The Galaxy UI import method might also work.  If you want to use that method:

Go to https://galaxy.ansible.com/my-content/namespaces

Select your namespace e.g. linux-system-roles

Find the repository you are interested in. On the right hand side of the window will be a button called Import. Click this button to start importing. Galaxy will import and create releases for each release under https://github.com/linux-system-roles/$ROLENAME/releases. Once the import is complete, go to the role's page on Galaxy e.g. https://galaxy.ansible.com/linux-system-roles/network. You should see your release version listed under Versions. If you do not, check the import log for errors.
```

Reply:
> We should change this section. I think the `Import` button only works if the role main branch name is `master`. I think we should say something like this:

Thank you, @richm! Updated the pr.